extra-dev/packages/coq-parsec/coq-parsec.dev/opam

```diff
@@ -20,6 +20,9 @@ depends: [
 ]
 
 tags: [
+  "category:CS/Data Types and Data Structures"
```

Comment:
The category is actually called `"category:Computer Science/Data Types and Data Structures"`. Yours will cause the creation of a duplicate category.